5calls/app/src/main/java/org/a5calls/android/a5calls/net/FiveCallsApi.java

```diff
@@ -345,6 +347,31 @@ public Map<String, String> getHeaders() throws AuthFailureError {
         mRequestQueue.add(request);
     }
 
+    public void reportSearch(String searchTerm) {
+        try {
+            JSONObject jsonBody = new JSONObject();
+            jsonBody.put("query", searchTerm);
+            
+            JsonObjectRequest request = new JsonObjectRequest(Request.Method.POST, SEARCH_TRACKING, jsonBody,
+                    new Response.Listener<JSONObject>() {
+                        @Override
+                        public void onResponse(JSONObject response) {
+                            // Search report successful - no action needed
+                        }
+                    }, new Response.ErrorListener() {
+                @Override
+                public void onErrorResponse(VolleyError error) {
```

Comment:
optional: the indentation is a bit weird here; I'd expect the @Override methods in the two listeners to have the same level of indenentation

Reply:
yeah, you're right, I did the auto formatter in android studio again and it looks better now